### DISCUSSION TITLE
2024-08-19_이슈생성 관련 질문

### DISCUSSION THREAD

**Discussion by tttghost** [tttghost/wheelmaker#43] [Q&A]
태영이의 레포에 놀러가서 보다보니 제가 이슈를 생성할 수 있는것을 발견하였습니다.
역시 쥐피티에게 물어본 결과 public 레포는 이슈생성자체가 된다고 하는군요..
보통 어떤 방식으로 public 레포에서 이슈생성을 막으셨는지 궁금해 여쭙니다..!

**Comment by jongfeel** [comment]
public repository는 create issue 됩니다. 막는 기능은 없용.
